src/svg/server-outline.svg

```diff
@@ -1,5 +1,5 @@
 <svg width="512" height="512" viewBox="0 0 512 512" fill="none" xmlns="http://www.w3.org/2000/svg">
-<ellipse cx="256" cy="112" rx="176" ry="80" stroke="black" stroke-width="32" stroke-linecap="round" stroke-linejoin="round"/>
-<path d="M432 112C432 112 432 376.055 432 400C432 444.183 353.202 480 256 480C158.798 480 80 444.183 80 400C80 378.169 80 112 80 112" stroke="black" stroke-width="32" stroke-linecap="round" stroke-linejoin="round"/>
-<path d="M432 256C432 300.183 353.202 336 256 336C158.798 336 80 300.183 80 256" stroke="black" stroke-width="32" stroke-linecap="round" stroke-linejoin="round"/>
+<ellipse cx="256" cy="112" rx="176" ry="80" stroke="currentColor" stroke-width="32" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M432 112C432 112 432 376.055 432 400C432 444.183 353.202 480 256 480C158.798 480 80 444.183 80 400C80 378.169 80 112 80 112" stroke="currentColor" stroke-width="32" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M432 256C432 300.183 353.202 336 256 336C158.798 336 80 300.183 80 256" stroke="currentColor" stroke-width="32" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
I think these should actually be reverted to `stroke="black"` to match the way the other icons work. Oddly, the outline icons don't change color when e.g. using the `color` prop.

Reply:
Wait so running `npm run build` and `npm start` in Ionicons doesn't show me what I'll see in prod? 😅 What steps do I need to follow to see what we'll see in prod?

Reply:
> Wait so running npm run build and npm start in Ionicons doesn't show me what I'll see in prod?

No 🙃 I've tried to find a good way to fix that, but I haven't quite figured it out yet.

>  What steps do I need to follow to see what we'll see in prod?

You could make a dev build and test it on a CodePen. Alternatively, you could run `npm run build` and look at the SVGs generated in `dist/svgs`.